website/content/terminal/futures/_index.md

```diff
@@ -1,9 +1,56 @@
 ---
 title: Introduction to Futures
-keywords: "forex, currency, money, hedge, dollar, euro"
-excerpt: "The Introduction to Futures explains how to use the 
-menu and provides a brief description of its sub-menus"
+keywords: "forex, currency, money, hedge, dollar, euro, futures, future, interest, rates, forwards, commodities, bonds, treasuries"
+excerpt: "This guide explains how to use the Futures menu and provides a brief description of its sub-menus"
 geekdocCollapseSection: true
 ---
+The Futures menu provides historical prices and the current term structure for an asset. This menu is accessible from the Main menu by typing `futures` and pressing the `enter` key. The available data is currently limited to what is publically available from the yFinance library. The feature set is small, but over time it will expand to include more markets and data.
```

Comment:
Let's not include that we offer less than the yFinance library or that the amount of features is limited.